backend/addcorpus/models.py

```diff
@@ -408,3 +397,35 @@ def clean(self):
                     'the corpus or correct the following errors.',
                     e
                 ])
+
+class CorpusDocumentationPage(models.Model):
+    class PageType(models.TextChoices):
```

Comment:
I didn't know this base class existed for choices. Much easier to read then the nested tuple business.